features/device-config/sink/consumer/src/main/java/org/opennms/features/deviceconfig/sink/consumer/DeviceConfigConsumer.java

```diff
@@ -78,9 +78,10 @@ public void handleMessage(DeviceConfigDTO message) {
             if (ipInterface != null) {
                 deviceConfigDao.updateDeviceConfigContent(
                         ipInterface,
-                        message.fileName,
+                        message.fileName, // use filename as config type
```

Comment:
`filename` was added to `DeviceConfig` object and database table, this has been merged to `feature/device-config`. Should get that code and add both `configType` and `fileName` here and elsewhere.

Reply:
In case of sink, we have nothing else than the filename. Maybe we need some extra configuration that maps filenames into config types. Or at least, we could allow to configure the config type to use. You guys will figure out what to do!

Reply:
Good point! Still, just want to make sure that the filename does get saved to the new `filename` field in the database, regardless of what we do about figuring out the config type.

Reply:
i think for Sink, we can make `config-type` as `default` for now and later we can check if we can derive it from filename itself.

Reply:
Use `ConfigType.Default` for now.